### ISSUE TITLE
README file

### ISSUE THREAD

**Issue by gabber12** [aanand/compose-file#36]
Can we have a readme listing down usage examples and contributing guide

**Comment by dnephin** [comment]
I don't think this repo is being maintained. You can find the current version of this code at https://github.com/docker/cli/tree/master/cli/compose
